torchtext/experimental/vocab.py

```diff
@@ -71,6 +71,23 @@ def vocab_from_file(file_object, min_freq=1, unk_token='<unk>', num_cpus=4):
     return Vocab(vocab_obj)
 
 
+def build_vocab_from_iterator(iterator):
```

Comment:
Any particular reason for not exposing the same arguments as `vocab` for `min_freq` and `unk_token`?

Reply:
There is an equivalent func in `torchtext.vocab` and we are maintaining the same behavior here. We could add `min_freq` and `unk_token` here.

Reply:
`min_freq` and `unk_token` are exposed in `load_vocab_from_file` and `load_vocab_from_raw_text_file` so does it not make sense to expose them (as well as `pad_token`) in `build_vocab_from_iterator`?

Reply:
I can add those two for sure.